src/emulator/extensions/utils.ts

```diff
@@ -0,0 +1,44 @@
+import * as planner from "../../deploy/extensions/planner";
```

Comment:
let's give this file a more specific name so it doesn't become a dump yard in the future..

Reply:
Renamed this to validation.ts, with the intention of using it as a place to separate out & unit test various functions we use to validate the users extensions and project before emulation